webextensions/api/downloads.json

```
@@ -1230,7 +1230,8 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "47"
+                "version_added": "47",
+                "notes": "Only the NETWORK_FAILED, FILE_FAILED, CRASH, USER_CANCELED, SERVER_BAD_CONTENT, SERVER_FORBIDDEN, SERVER_UNAUTHORIZED, and SERVER_FAILED errors are returned."
```

Nit: Let's put the keywords in code fences:

```suggestion
                "notes": "Only the `NETWORK_FAILED`, `FILE_FAILED`, `CRASH`, `USER_CANCELED`, `SERVER_BAD_CONTENT`, `SERVER_FORBIDDEN`, `SERVER_UNAUTHORIZED`, and `SERVER_FAILED` errors are returned."
```

---

To make it more readable:

```suggestion
                "notes": "Only returns the following errors: `NETWORK_FAILED`, `FILE_FAILED`, `CRASH`, `USER_CANCELED`, `SERVER_BAD_CONTENT`, `SERVER_FORBIDDEN`, `SERVER_UNAUTHORIZED`, and `SERVER_FAILED`"
```

---

@claudepache to improve further:
```suggestion
                "notes": "Only returns these errors: `NETWORK_FAILED`, `FILE_FAILED`, `CRASH`, `USER_CANCELED`, `SERVER_BAD_CONTENT`, `SERVER_FORBIDDEN`, `SERVER_UNAUTHORIZED`, and `SERVER_FAILED`"
```